src/main/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPC.java

```diff
@@ -130,7 +130,7 @@ protected static int getCurrentTxPower(JsonArray latestDeviceStatus, String band
 				return radioObject.get("tx-power").getAsInt();
 			}
 		}
-		return 0;
```

Comment:
I made this change to differentiate an AP that can operate on the band but just happens to be transmitting at 0 dBm vs an AP that cannot operate on the band